tests/indexes/test_datetime_index.py

```diff
@@ -35,18 +35,6 @@ def test_index_relops() -> None:
     check(assert_type(data[idx >= x], pd.DatetimeIndex), pd.DatetimeIndex)
     check(assert_type(data[idx > x], pd.DatetimeIndex), pd.DatetimeIndex)
 
-    # TODO: https://github.com/pandas-dev/pandas-stubs/pull/1438#discussion_r2451864012
```

Comment:
Hey @harshasiddartha, since you are deleting some tests can you point out where the other tests are so we make sure that we maintain a good test coverage on the project? Thanks!